test/toplevel.jl

```diff
@@ -540,3 +540,15 @@ end
     modexs = collect(ExprSplitter(@__MODULE__, ex))
     @test length(modexs) == 3
 end
+
+@testset "toplevel global" begin
+    ex = :(begin
+        global foo_g = 10
+        sin(foo_g)
+    end)
+    modexs = collect(ExprSplitter(@__MODULE__, ex))
+    for (mod, ex) in modexs
+        @test JuliaInterpreter.finish!(Frame(mod, ex), true) === nothing
+    end
+    @test length(modexs) == 1
+end
```

Comment:
```suggestion
@testset "toplevel scope annotation" begin
    ex = Base.parse_input_line("""
    global foo_g = 10
    sin(foo_g)
    """)
    modexs = collect(ExprSplitter(@__MODULE__, ex))
    for (mod, ex) in modexs
        @test JuliaInterpreter.finish!(Frame(mod, ex), true) === nothing
    end
    @test length(modexs) == 2

    ex = Base.parse_input_line("""
    local foo = 10
    sin(42)
    """)
    modexs = collect(ExprSplitter(@__MODULE__, ex))
    for (mod, ex) in modexs
        @test JuliaInterpreter.finish!(Frame(mod, ex), true) === nothing
    end
    @test length(modexs) == 2
end
```
since the previous version passes on master.